README.md

```diff
@@ -194,7 +196,42 @@ Done with this data source? Remove it with the following command:
 ```sql
 DROP DATABASE my_pg;
 ```
-
+## Supported data sources
+
+| Source                 | Read | Write | Table Function | External Table | External Database | Supported Object Stores             |
+| ---------------------- | :--: | :---: | :------------: | :------------: | ----------------- | ----------------------------------- |
+| **Databases**          |  --  |  --   |       --       |       --       | --                | --                                  |
+| MySQL                  |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| PostgreSQL             |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Microsoft SQL Server   |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| MongoDB                |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Snowflake              |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| BigQuery               |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
+| Cassandra/ScyllaDB     |  ✅  |  ✅   |       ✅       |       ✅       | ✅                | ➖                                  |
```

Comment:
https://github.com/GlareDB/glaredb/issues/1143 I don't think we support writes/inserts for many of these case (and def not for mongodb.)